packages/cloudscape-react-ts-website/src/cloudscape-react-ts-website-project.ts

```diff
@@ -129,10 +155,24 @@ export class CloudscapeReactTsWebsiteProject extends ReactTypeScriptProject {
       hooksPackage:
         tsApi.library?.typescriptReactQueryHooks?.package?.packageName,
     }));
+    const webSocketApis = this.typeSafeWebSocketApis.map((tsApi, idx) => ({
+      apiName: tsApi.model.apiName,
+      isLast: idx === this.typeSafeWebSocketApis!.length - 1,
+      apiNameSafe: tsApi.model.apiName
+        ?.replace(/[^a-z0-9_]+/gi, "")
+        .replace(/^[0-9]+/gi, ""),
+      hooksPackage:
+        tsApi.library?.typescriptWebSocketHooks?.package?.packageName,
+      clientPackage:
+        tsApi.library?.typescriptWebSocketClient?.package?.packageName,
+    }));
     const mustacheConfig = {
       applicationName: this.applicationName,
+      hasApis: apis.length + webSocketApis.length > 0,
```

Comment:
TILI: rename apis to typeSafeApis

Reply:
Might leave it - we already have `this.typeSafeApis` which are the project instances, and then `apis` and `webSocketApis` are the mustache configs for each :)